main.py

```diff
@@ -1,10 +1,46 @@
 import init_django_orm  # noqa: F401
+import json
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json", "r") as file:
+        players_data = json.load(file)
+    for nickname, player_data in players_data.items():
+        # Create or get race
+        race_data = player_data["race"]
+        race, _ = Race.objects.get_or_create(
+            name=race_data["name"],
+            defaults={"description": race_data.get("description", "")}
+        )
+
+        # Create or get skills for the race
+        for skill_data in race_data.get("skills", []):
+            Skill.objects.get_or_create(
+                name=skill_data["name"],
+                defaults={"bonus": skill_data["bonus"], "race": race}
+            )
+
+        # Create or get guild (if exists)
+        guild_data = player_data.get("guild")
+        guild = None
+        if guild_data:
+            guild, _ = Guild.objects.get_or_create(
+                name=guild_data["name"],
+                defaults={"description": guild_data.get("description")}
+            )
+
+        # Create player
+        Player.objects.get_or_create(
+            nickname=nickname,
+            defaults={
+                "email": player_data["email"],
+                "bio": player_data["bio"],
+                "race": race,
+                "guild": guild
```

Comment:
The 'guild' field in the Player model is nullable. Ensure that the 'defaults' dictionary correctly assigns 'None' to 'guild' when 'guild_data' is not provided, which is currently handled correctly.